piptools/scripts/options.py

```diff
@@ -364,3 +371,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_targets",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_TARGETS),
+    help="Name of a build target to install dependencies for. "
+    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
+    "well; may be used more than once.",
```

Comment:
Can we replace "included as well" with "included implicitly + additionally" in all the options here?

Reply:
Are you suggesting it should read _Additionally, static dependencies declared in 'pyproject.toml::build-system.requires' will be included implicitly_?

Reply:
@apljungquist yes, that's about what I wanted. I did not have a few precise wording in mind but what you composed makes sense to me.

Reply:
Can we mark this as resolved then?

Reply:
Sure, marked as resolved.